phing/build.yml

```diff
@@ -58,9 +60,9 @@ drupal:
 
 drush:
   bin: ${composer.bin}/drush
-  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${multisite.name}
+  cmd: ${drush.bin} @${drush.alias} -r ${docroot} -l ${site.uri.${environment}}
```

Comment:
@danepowell - #1023 is a swipe at breaking it up and reducing the scope to just setup tasks, but at this point it keeps the `uri` argument in place but this PR is probably the best place to begin phasing its use out.